packages/devtools_app/lib/src/screens/memory/memory_heap_tree_view.dart

```diff
@@ -454,6 +473,9 @@ class HeapTreeViewState extends State<HeapTree>
                     ],
                   ),
                 ),
+
+                // Leaks tab.
+                if (controller.shouldShowLeaksTab.value) const LeaksPane(),
```

Comment:
Wrap this in KeepAliveWrapper so that the LeaksPane is not completely destroyed and rebuilt on tab switches.